src/app/(school)/page.tsx

```diff
@@ -16,6 +16,13 @@ export default function Home() {
     universityId: 1,
     courseId: "2a45bab1-5ec4-4d2e-b245-27a142a78890",
   });
+  const publicCourses = api.courses.listByUni.useQuery({
+    universityId: 1,
```

Comment:
Keeping track of all university IDs might be difficult for development. Do you thinking a university-id mapping will help? For example a configuration file that specifies `SMU: 1`, and in code it will be `config.SMU` etc

Reply:
I agree that just using IDs would be anonying to have, but I don't think it's practical to keep a config like that :/ for eg, if the database migrates, and the uni id changes, we'll need to reconfigure the app in order to read from the database

How I expect this endpoint to be used is - in conjunction with other APIs like `api.university.getAll`, where the returned uni id is kept on the breadcrumb uni select button as the key-value pair. When the user selects the uni, it's corresponding id will be used to query for the course page (passed as props)

A better alternative to this would probably to think of a better way to maintain a cache of such data in the FE so we dont have to make that many db requests to data that is predictably unchanging

Reply:
Hi @davidlhw, thanks for the clarifications :)) I would agree with a caching strategy since there might be cases where we wont be calling `university` endpoint before accessing certain content. We can keep this to future discussion. Feel free to resolve the comment!

Reply:
opened #102 to track this!

will be resolving this comment